src/components/browse/sidebar/filter.tsx

```diff
@@ -39,30 +41,33 @@ const Filter = (props: filterProps) => {
           .sort((a, b) => (b.option < a.option ? 1 : -1))
           .map((option) => {
             let key: string = props.facet.key + ":" + option.option;
-            return (
-              <div className="p-1 d-flex align-top" key={key}>
-                <Form.Check
-                  id={key}
-                  className="d-inline-block"
-                  checked={props.check.get(key)}
-                  value={key}
-                  onChange={(event) => handleCheck(key, event)}
-                  style={{ zIndex: 100 }}
-                />
-                <Form.Label className="p-0 m-0 w-100" htmlFor={key}>
-                  <Row>
-                    <Col lg md sm xl xs xxl={10}>
-                      <p className="ps-2 my-0">{option.option}</p>
-                    </Col>
-                    <Col lg md sm xl xs xxl={2} className="h-100">
-                      <Form.Label className="p-0 m-0" htmlFor={key}>
-                        {option.count}
-                      </Form.Label>
-                    </Col>
-                  </Row>
-                </Form.Label>
-              </div>
-            );
+            if (option.option.includes(",") !== true) {
```

Comment:
The response from the backend has 11 facets for the "type" and in the UI we are now just showing 7 facets. I dont think we can ignore the facets that has ","

Reply:
Yes, because all the facets that had a comma were facets that were there because the Dataset had multiple study types. Deepak said he was going to remove returning these at some point because they are completely redundant. If you look at dev, you will notice that all facets that have a comma are already present as facets without commas.

Reply:
Yes, but this is a fix that needs to come from the backend. We dont have to do it. We display the facets as we receive.

Reply:
Ok